cmd/goal/network.go

```diff
@@ -101,7 +122,7 @@ var networkCreateCmd = &cobra.Command{
 			consensus, _ = config.PreloadConfigurableConsensusProtocols(dataDir)
 		}
 
-		network, err := netdeploy.CreateNetworkFromTemplate(networkName, networkRootDir, networkTemplateFile, binDir, !noImportKeys, nil, consensus, devModeOverride)
+		network, err := netdeploy.CreateNetworkFromTemplate(networkName, networkRootDir, networkTemplateFile, defaultReader, binDir, !noImportKeys, nil, consensus, devModeOverride)
```

Comment:
You can open the file and use the reader for both cases instead of supporting a file or a reader.

Reply:
So we'd be deploying the json file along with the binary?

Reply:
Since the io.Reader and a file are two ways of getting the same data, I was suggesting you could pick one instead of offering both.

Reply:
I understand. But then that json file would have to reside somewhere. Seems neater to do it this way.

Reply:
This suggestion is really a nitpick, so you can take it or leave it. It seems like there is a misunderstanding here so I figure I'll show what I meant: https://github.com/mciccarello/go-algorand/pull/1/files

Reply:
Oh, OK I see what you're saying now. Yeah, that code a slightly cleaner. I took your comments to mean read the json file rather then embed it.